lms/djangoapps/instructor_analytics/csvs.py

```diff
@@ -28,6 +30,9 @@ def create_csv_response(filename, header, datarows):
         quotechar='"',
         quoting=csv.QUOTE_ALL)
 
+    disclaimer = SensitiveMessageOnReports()
+    send_disclaimer = disclaimer.csv_direct(csvwriter)
+
     encoded_header = [unicode(s).encode('utf-8') for s in header]
```

Comment:
I have to edit this file since the `Download CSV of Certificates Issued` button build the CSV in this file, although has the same logic of the other CSV builders.